lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -433,7 +434,7 @@ def _get_job_reqs_from_concierge_params(
             cpus=norm.get(REQUEST_CPUS),
             memory_MB=norm.get(REQUEST_MEMORY),
             disk_GB=norm.get(REQUEST_DISK),
-            client_group=norm.get(CLIENT_GROUP),
+            client_group=norm.get(CLIENT_GROUP) or CONCIERGE_CLIENTGROUP,
```

Comment:
Is it noted anywhere that the default clientgroup is "concierge" for running concierge jobs? I'm assuming so.

Reply:
It's specified in the `deploy.cfg` file. The spec specifies that there's a default but not the exact name of the default, which I think is OK. Long term I'd probably want to make it configurable but I think there's a non-zero chance the changes I'm making will make `run_job_concierge` obsolete.